<launch>
	<!-- Start find mates node -->
	<node name="receptionist_node" pkg="frasier_receptionist" type="receptionist.py" output='screen'>
		<param name="use_hsr" type="bool" value="true" />
    	<remap from="~input_image_topic" to="/hsrb/head_rgbd_sensor/rgb/image_rect_color" />
	</node>
</launch>